infra/tf/locals.tf

```diff
@@ -0,0 +1,16 @@
+################################################################################
+# ZQ2 GCP Terraform locals variables
+################################################################################
+
+locals {
+  binary_location      = "${path.module}/../../target/x86_64-unknown-linux-gnu/release/zilliqa"
+  labels               = merge(var.labels, { "zq2-network" = var.network_name })
+  devnet_resources     = var.zq2_type == "devnet" ? 1 : 0
```

Comment:
What is this variable doing? It seems to conditionalise a whole bunch of resources, but I don't really understand why we want those resources conditionalised and how it relates to the devnet?

Reply:
I kept it for backward compatibility on how the existing devnet works. It provision VPCs, subnet and also gives people the possibility to generate random secrets without dependencies from 1password, so you can use this code to deploy dummy network avoiding the "path-to-prod" dependencies we have specifically on our environment and you still have the option to pass the options as variable inputs if needed (like secret keys or subnets). We can just remove this and add a skip_<resource_name> var instead or just have mandatory inputs for almost all the other stuff and simplify the code removing vpc/sn and so on. Happy both ways, no special need for me except keep the process which was in place working